### DISCUSSION TITLE
Problem Database Connection Error: 28P01 undefined:undefined after upgrade PostgreSQL

### DISCUSSION THREAD

**Discussion by dima-online** [requarks/wiki#5748] [Guides]
Hi
I installed the system as indicated on the page [https://docs.requarks.io/install/ubuntu](https://docs.requarks.io/install/ubuntu) (see Setup Containers)
The documentation says to install PostgreSQL 11. 
However, time goes by the database server version is very outdated.This is what the system told me.
I found an error after updating PostgreSQL to version 14:
_Database Connection Error: 28P01 undefined:undefined_
On the PostgreSQL server side:
_FATAL:  password authentication failed for user "wiki"
DETAIL:  Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"_

The reason is that the new version of PostgreSQL changed the file **/var/lib/postgresql/data/pg_hba.conf**
the scram-sha-256 method is in this file instead of md5:
`host all all all scram-sha-256`

Need to return the MD5 method at least for wiki container:
```
host    wiki            wiki            172.18.0.0/16           trust
host    wiki            wiki            all                     md5

host all all all scram-sha-256
```
Please add this clarification to the documentation

**Comment by binarywang** [comment]
I got this：
```
2025-01-21 10:19:17.310 GMT [24181] FATAL:  password authentication failed for user "postgres"
2025-01-21 10:19:17.310 GMT [24181] DETAIL:  Password does not match for user "postgres".
	Connection matched pg_hba.conf line 1: "host     all             all             0.0.0.0/0               md5"
```
